crates/sync/anvil/src/tracker.rs

```diff
@@ -1,5 +1,6 @@
-use std::{sync::Arc, time::Duration};
+use std::{ops::Add, sync::Arc, time::Duration};
 
+use base64::{engine::general_purpose, Engine as _};
```

Comment:
this should just be 

```
use base64::{self, Engine as _};
```

otherwise the call to `general_purpose` further down is hard to understand, since `general_purpose` by itself explains nothing

Reply:
Referring to [this](https://docs.rs/base64/latest/base64/#using-predefined-engines) here since this seems to be the new recommended way to decode `base64`.

Reply:
I think you're reading too much into that snippet. it's by no means a recommendation on how to set the `use` statement.
The functionality is exactly the same, this is purely a matter of readability.

And in anything other than a small snippet, having a `general_purpose` keyword dangling in the code is just plain confusing, since one would have to jump up to the imports to understand where it came from